code/modules/surgery/organs/eyes.dm

```diff
@@ -17,6 +17,11 @@
 	var/see_invisible = SEE_INVISIBLE_LIVING
 	var/lighting_alpha = LIGHTING_PLANE_ALPHA_VISIBLE
 
+/obj/item/organ/internal/eyes/Initialize(mapload, ...)
+	. = ..()
+	if(!eye_colour)
```

Comment:
If all the eyes have a var/eye_colour set to #000000 by default, is this needed?

Reply:
It's not but I thought I'd add it seeing how the issue already was caused by somebody forgetting that it couldn't be null.
I'm myself still on the fence for this one though since it's only here to prevent coding mistakes (which prob will happen)

Reply:
Future coding mistakes are as likely to happen with eye_colour being null as, say, colourblind_matrix being null. If you argue that this check is useful, then you have to admit that this is only a half-measure, and you really need to set *all* the fields to their default values (and do that pretty much for every object). I hope you would agree that is overkill, since you *already* have default values set in the object definition.

Reply:
Changed. eye_colour I can see getting set to null. But having this check promotes lazy/non_tested coding which I don't like. Also sets the wrong expectation seeing you can set it to null on init